eos/parameters/hadron-properties.yaml

```diff
@@ -934,3 +934,64 @@ groups:
           latex:   '$\rho_D^3(1\,\GeV)$'
           unit:    'GeV^2'
           comment: 'cf. [MvD:2016A]'
+
+
+  # Parameters of the ee -> ccbar scattering processes
+  # {{{
+  - title: 'Parameters of the ee -> ccbar scattering processes'
+    description: ''
+    parameters:
+      # Non-ccbar component of R
+      'ee->ccbar::Rconstant' :
+          central: 0
+          min:     0
+          max:     0
+          latex:   '$R_{uds}$'
+          comment: ''
+      # Normalization of R
+      'ee->ccbar::exclusive_norm' :
+          central: 1.0
+          min:     0.5
+          max:     1.5
+          latex:   '$N_\textrm{excl}$'
+          comment: ''
+
+      # charmonimum resonances size
+      'size::%1%' :
```

Comment:
Hm, this is the wrong way around, isn't it? ``psi(2S)::q_R``? Also, I prefer to use ``q_R`` rather than size I think.